src/core/clean/mod.rs

```diff
@@ -270,26 +291,42 @@ impl TryInto<TopLevelStatement> for WithSrc<py::Statement> {
                             symbol,
                             alias: None,
                         }),
-                        _ => Err(Error::ImportAlias.core(location.clone())),
+                        _ => Ok(ImportSymbol {
+                            symbol,
+                            alias,
+                        }),
                     })
                     .collect::<Result<_, _>>()?,
             }),
             py::StatementType::Assign { targets, value } => {
-                if targets.len() != 1 {
+                /*if targets.len() != 1 {
                     Err(Error::InvalidConstant)
-                } else {
+                } else */{
```

Comment:
Worth just deleting the dead code, we can get it back from git history if we need it, and the next dev will wonder why it's commented out. This isn't a PR review (I'm new to Rust) just as a general ideal.